jsonnet/telemeter/rules.libsonnet

```diff
@@ -35,6 +35,29 @@
                 topk by (_id) (1, max by (_id, managed, ebs_account, internal) (label_replace(label_replace((subscription_labels{support=~"Standard|Premium|Layered"} * 0 + 1) or subscription_labels * 0, "internal", "true", "email_domain", "redhat.com|(.*\\.|^)ibm.com"), "managed", "", "managed", "false")) + on(_id) group_left(version) (topk by (_id) (1, 0*cluster_version{type="current"})))
               |||,
             },
+            {
+              record: 'subscription_labels:not_redhat_not_ibm',
```

Comment:
I don't think we need this because we have `id_version_ebs_account_internal:cluster_subscribed`

Reply:
agreed, closing